packages/manager/core/generator/app/templates/src/components/ManagerTile/ManagerTile.scss

```diff
@@ -0,0 +1,28 @@
+.manager-tile {
+  .tile {
+    width: 100%;
+
+    .tile-type {
+      display: block;
+      margin-bottom: 16px;
+    }
+
+    .tile-title {
+      display: block;
+      margin-bottom: 16px;
+    }
+
+    .tile-description {
+      display: block;
+      margin-bottom: 12px;
+    }
+
+    a {
+      text-decoration: none;
+    }
+
+    .link-icon {
+      margin-left: 10px;
```

Comment:
**intpick:** Use bootstrap utility classes instead

Reply:
we will delete this component with the supercomponent when it will be merged